.github/workflows/format_fix.yml

```diff
@@ -1,4 +1,4 @@
-name: Check for formatting errors
+name: Automatic Formatting on PRs
```

Comment:
You're going to want to follow the "fix" with a "check". Not everything can always be fixed automatically and formatting errors should be caught before the PR is merged. In fact, that's the DoD for this whole initiative.

Reply:
Sure

Reply:
Done in 715a8e096c4022c5cddc560ed0ebe1fbfc06c778